helpers/lbryHelpers.js

```diff
@@ -1,6 +1,7 @@
 var path = require('path');
```

Comment:
`lbryHelpers.js` sounds like and sort of looks like the kind of file that is on track to accumulate a lot of functions and become a mess. Worth looking at the typical patterns for unique or business-level modal functions in this framework.